src/static/graphkb-env-config.js

```diff
@@ -9,4 +7,5 @@ window._env_ = {
   CONTACT_EMAIL: 'graphkb@bcgsc.ca',
   CONTACT_TICKET_URL: 'https://www.bcgsc.ca/jira/projects/KBDEV',
   PUBLIC_PATH: '/',
+  IS_DEMO: '',
```

Comment:
accidental commit?

Reply:
yes, shoot. this file gets created every time you run the app and I'm constants having to revert it

Reply:
fixed in [edb87f9](https://github.com/bcgsc/pori_graphkb_client/pull/51/commits/edb87f9b15f38bb438ce748a51586105f00b3773)